binr/ragg2/ragg2.c

```diff
@@ -468,17 +469,13 @@ int main(int argc, char **argv) {
 				eprintf ("No format specified wtf\n");
 				goto fail;
 			}
+			RPrint *p = r_print_new();
```

Comment:
Please put the space between function name and parentheses.

Reply:
Thanks for your review. It is now fixed and rebased.